syntax = "proto3";

package contracts.services.gateway.users;

import "contracts/services/users/user.proto";

message CreateUserRequest {
  string email = 1;
  string last_name = 2;
  string first_name = 3;
  string middle_name = 4;
  string phone_number = 5;
}

message CreateUserResponse {
  contracts.services.users.User user = 1;
}